src/org/mozilla/javascript/typedarrays/NativeTypedArrayView.java

```diff
@@ -237,6 +238,19 @@ private void setRange(NativeArray a, int off) {
         }
     }
 
+    private static Object getElem(Context cx, Scriptable target, long index) {
```

Comment:
TypedArray cannot accept an ArrayLike, so it probably doesn't need `getElem` and `getRawElem`.
https://262.ecma-international.org/13.0/#sec-%typedarray%.prototype.at
> 2. 2. Perform ? [ValidateTypedArray](https://262.ecma-international.org/13.0/#sec-validatetypedarray)(O).

Reply:
I pushed code removing the getElems functions and returning to getProperty. Please let me know what you think.

Reply:
I think it looks good. Thanks!